confluence/models/page.py

```diff
@@ -57,5 +57,15 @@ def __init__(self, json):  # type: (Dict[str, Any]) -> None
             if 'lastUpdated' in json['history']:
                 self.last_updated = PageUpdate(json['history']['lastUpdated'])
 
+        if 'version' in json:
+            if 'number' in json['version']:
+                self.version_number = json['version']['number']
+
+    def for_page_update(self) -> Dict[str, Any]:
```

Comment:
I'm not sure about this living on the page object, it's really content type independent so I'd rather not abstract it out of the put content function until necessary. I'd suggest changing the API to put_content to be:
(id: str, new_version: int, content: str, status: Optional[ContentStatus]=None, conflict_policy: Optional[ConflictPolicy]=None)

and construct the json object in that function. If and when there's commonality elsewhere it can be moved.